x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/aggregation/blockhash/BytesRef3BlockHash.java

```diff
@@ -137,7 +137,7 @@ void add() {
                 }
                 finishMv();
             }
-            emitOrds();
+            flushRemaining();
```

Comment:
I could have made this come naturally when you `close()` the thing, but I felt like that'd be confusing.